python/agents/medical-pre-authorization/medical_pre_authorization/subagents/data_analyst/prompt.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+sample_report="""
+*****************************Sample Report Document Template***********
+Medical Pre-Authorization Request Report
+Date of Report: July 19, 2025
+
+Pre-Authorization for Elective LASIK Surgery (Rejection)
+Patient Name: Mark Johnson
+
+Treatment Name: Bilateral LASIK Eye Surgery
+
+Insurance Provider: OptiCare Health Plans
+
+Summary of Medical Records:
+Patient Mark Johnson seeks LASIK surgery for refractive error correction to reduce dependence on eyeglasses. Corrected visual 
+acuity is 20/25 in both eyes. No medical contraindications to continued use of corrective lenses or severe uncorrectable refractive error are noted.
+
+Summary of Insurance Coverage:
+OptiCare Health Plans policy (Policy ID: LASIK-ELEC-MJ-2024) was reviewed. The policy explicitly categorizes LASIK as an 
+elective/cosmetic procedure. Coverage is provided only in cases of severe refractive error (e.g., >7.5 diopters) or documented 
+medical intolerance to traditional corrective lenses (glasses/contacts). Neither of these criteria is met.
+
+Pre-Authorization Claim Decision: REJECTED
+Reason for Rejection: The requested procedure (Bilateral LASIK Eye Surgery) is categorized as an elective cosmetic procedure under 
+the patient's OptiCare Health Plans policy and does not meet the strict medical necessity criteria for coverage.
+"""
+
+def return_instructions_data_analyst():
```

Comment:
again, not sure why we need a function here...? We should just use a constant...